pt-BR/active_record_validations.md

```diff
@@ -1028,18 +1028,18 @@ end
 A validação é executada apenas quando todas as condições `:if`
 e nenhuma das condições `:unless` resultarem em `true`.
 
-Performing Custom Validations
------------------------------
+Realizando Validações Customizadas
+----------------------------------
 
-When the built-in validation helpers are not enough for your needs, you can
-write your own validators or validation methods as you prefer.
+Quando as validações auxiliares embutidas não são o bastante para suas necessidades, você pode
```

Comment:
```suggestion
Quando os *helpers* de validação embutidos não são o bastante para suas necessidades, você pode
```

No [Guia de Tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md) fala mais sobre isso mas a palavra *helper* ligada a código a gente costuma só colocar em itálico por ser um termo técnico, pode dar uma olhada?